### PR TITLE
Asim/fix network parsers

### PR DESCRIPTION
**Pull request by oshezaf**

### CODE REVIEW

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftLinuxSysmon.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: ASIM Sysmon for Linux Network Session Parser
-  Version: '0.0'
-  LastUpdated: Sep 9th, 2021
+  Version: '0.2'
```
Comment (yaronMSFT) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftLinuxSysmon.yaml:3]: what happened to 0.1? :-)

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftLinuxSysmon.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: ASIM Sysmon for Linux Network Session Parser
-  Version: '0.0'
-  LastUpdated: Sep 9th, 2021
+  Version: '0.2'
```
Comment (yaronMSFT) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftLinuxSysmon.yaml:3]: 0.1?

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoft365Defender.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: M365 Defender Network Sessions
-  Version: '0.1'
-  LastUpdated: Aug 4, 2021
+  Version: '0.2'
```
Comment (yaronMSFT) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoft365Defender.yaml:3]: Although no direct change here.

Comment (oshezaf) [reply Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoft365Defender.yaml:3]: As said on the web parsers. Point taken, but for this iteration I will keep this like this.

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftLinuxSysmon.yaml (hunk 2 above)
Comment (oshezaf) [reply Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftLinuxSysmon.yaml:3]: Same

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftWindowsEventFirewall.yaml [hunk 4]
```diff
@@ -77,7 +77,7 @@ ParserQuery: |
         '\x0d\x0a  <Data Name="Application">'Application'</Data>'
         '\x0d\x0a  <Data Name="Direction">'DirectionCode'</Data>'
         '\x0d\x0a  <Data Name="SourceAddress">'SourceAddress'</Data>'
-        '\x0d\x0a  <Data Name="SourcePort">'SourcePort'</Data>'
+        '\x0d\x0a  <Data Name="SourcePort">'SourcePort:int'</Data>'
         '\x0d\x0a  <Data Name="DestAddress">'DestAddress'</Data>'
```
Comment (YaronFruchtmann) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftWindowsEventFirewall.yaml:81]: is the default string? maybe we should cast explicitly?

**Review by YaronFruchtmann — APPROVED** [review]
I left a comment about explicit casting, but it is not a must

**Pull request merged**